libs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@audius/libs",
-  "version": "1.2.106",
+  "version": "1.2.107",
```

Comment:
I think someone else is also touching the libs version
Generally, we like to do this as a separate, independent PR

Reply:
moved this out - thanks for callout